go.mod

```diff
@@ -1,98 +1,195 @@
 module github.com/kube-logging/telemetry-controller
 
-go 1.21
+go 1.22
 
-toolchain go1.21.5
+toolchain go1.22.1
 
 require (
 	emperror.dev/errors v0.8.1
 	github.com/cisco-open/operator-tools v0.34.0
-	github.com/onsi/ginkgo/v2 v2.13.0
-	github.com/onsi/gomega v1.29.0
-	github.com/open-telemetry/opentelemetry-operator v0.90.0
-	k8s.io/apimachinery v0.28.4
-	k8s.io/client-go v0.28.4
-	sigs.k8s.io/controller-runtime v0.16.3
+	github.com/onsi/ginkgo/v2 v2.16.0
+	github.com/onsi/gomega v1.31.1
+	github.com/open-telemetry/opentelemetry-operator v0.96.0
```

Comment:
Because of this I think `ENVTEST_OTEL_OPERATOR_VERSION` in the Makefile should be bumped as well.